flutter/lib/resources/config_manager.dart

```diff
@@ -63,12 +66,22 @@ class ConfigManager {
 
   Future<File> createConfigListFile() async {
```

Comment:
I think name `updateConfigListFile` would be more suitable with this change.
Also we can make this function private. The only external use for it in the `BenchmarkState.create` is useless.

Reply:
Done